src/lib/country-data.js

```diff
@@ -12,6 +12,32 @@ module.exports.countryOptions = Object.keys(countries).map(code => ({
         return 1;
     });
 
+const overrideOptions = module.exports.overrideOptions = (countryNameOverrides, thisCountryOptions) => {
```

Comment:
At first glance, it is hard for me to tell how to use these. i.e. What kind of object/list do I pass as countryOverrides?
Also, What does "thisCountryOptions" mean?  Similar question below about commonCountryCodes and thisCountryOptions. It looks like the thisCountryOptions is optional?  Could you add some documentation?

Reply:
i can give them comments explaining what they do.

Reply:
Done